src/bartholomew.rs

```diff
@@ -46,8 +46,8 @@ pub fn render(req: Request) -> Result<Response> {
 
     let base_url = std::env::var(BASE_URL_ENV);
 
-    if let Ok(..) = base_url {
-        config.base_url = Some(base_url.unwrap());
+    if let Ok(bb) = base_url {
```

Comment:
Related to this changeset?  Or accidentally added?

Reply:
Oh for some reason the build failed if I didn't change it.  I think maybe there is a new lint or something in the new Rust version and Bartholomew isn't pinned?  Not sure.  I can back it out and we can see what happens if you want.  (I should at least give the variable a better name.)

Reply:
Ah ok 👍